admin/commands/management/remove.go

```diff
@@ -45,6 +48,42 @@ type removeMySQLCommand struct {
 }
 
 func (cmd *removeMySQLCommand) Run() (commands.Result, error) {
+	status, err := agentlocal.GetStatus(agentlocal.DoNotRequestNetworkInfo)
+	if err != nil {
+		return nil, err
+	}
+
+	if cmd.ServiceID == "" && cmd.ServiceName == "" {
+		// Automatic service lookup during removal
+		//
+		// Get services and remove it automatically once it's only one
+		// service registered
+		servicesRes, err := inventoryClient.Default.Services.ListServices(&services.ListServicesParams{
```

Comment:
Is it possible that node hasn't any services?

Reply:
So we will invoke `RemoveService` with empty parameters in that case? User will get descriptive error in that case?

Reply:
`service_id or service_name expected` message returns from the `managed` and right now it returns from the two different cases

1. We have more than one service registered on a node
2. We have no services registered on a node.

I think that I'll edit the returning message to something meaningful

Reply:
I just changed the default behavior of `managed` validators. Now it returns more user-friendly errors.